doozerlib/build_status_detector.py

```diff
@@ -20,44 +20,27 @@ def __init__(self, session: ClientSession, logger: Optional[Logger] = None):
         self.shipping_statuses: Dict[int, bool] = {}  # a dict for caching build shipping statues. key is build id, value is True if shipped.
         self.archive_lists: Dict[int, List[Dict]] = {}  # a dict for caching archive lists. key is build id, value is a list of archives associated with that build.
 
-    def find_embargoed_builds(self, builds: List[Dict]) -> Set[int]:
-        """ find embargoes in given list of koji builds
+    def find_embargoed_builds(self, builds: List[Dict], candidate_tags: List[str]) -> Set[int]:
```

Comment:
Thanks, I missed that. Unit tests didn't stop me ;-)

That's inconvenient because we really need to know the embargoed tag in order to determine if an image contains RPMs tagged with it, and `detect-embargo nvr` doesn't supply candidate tags yet. I think I'll expand the required parameters to that cmd; I don't think we have any automation around it yet.

Reply:
Saw a better way hopefully.